web/pdf_presentation_mode.js

```diff
@@ -342,18 +358,86 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
     /**
      * @private
      */
+    _touchSwipeState: null,
```

Comment:
This is usually not how we initialize member variables. So let's just place `this._touchSwipeState = null:` inside of ` function PDFPresentationMode(options) {` instead (e.g. after line [pdf_presentation_mode.js#L64](https://github.com/staktrace/pdf.js/blob/326610a903386da431fc540b5a94c99b697eee5c/web/pdf_presentation_mode.js#L64)).